php/savesettings.php

```diff
@@ -48,6 +48,36 @@ function validDomain($domain_name)
 			"8.20.247.20" => "Comodo"
 		];
 
+$adlistsdefault = [];
+$adlistsuser = [];
+function readAdlists(&$list, $listname)
+{
+	// Reset list
+	$list = [];
+	$handle = @fopen("/etc/pihole/adlists.".$listname, "r");
+	if ($handle)
+	{
+		while (($line = fgets($handle)) !== false)
+		{
+			if(substr($line, 0, 2) === "#h")
```

Comment:
Could this check for `#http` instead? There might be a comment that starts like `#here's some list...`